spacy/tokens/token.pyx

```diff
@@ -787,8 +791,7 @@ cdef class Token:
 
         RETURNS (str): IOB code of named entity tag.
         """
-        iob_strings = ("", "I", "O", "B")
-        return iob_strings[self.c.ent_iob]
+        return self.iob_strings()[self.c.ent_iob]
```

Comment:
This stuff is kind of frustrating every time. Maybe we should have a `token.iob_tag` attribute?

Reply:
I'm not sure I understand what you mean? `token.ent_iob_` does give the IOB tag. But I guess the question is, who should be responsible of making the conversion from int to string?

Reply:
I just meant it's annoying to assemble the tag from the IOB component and the ent_type component. It'd be kind of nice to have an attribute that just did it.